submissions/pashapushak1996/html-css-popup/index.html

```diff
@@ -0,0 +1,204 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Uchen&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/main.css">
+    <title>Popup</title>
+</head>
+<body>
+<div class="wrapper">
```

Comment:
Please make indentations less, to make it more readable

Reply:
I solve this. Thanks